domains/mlsend2._domainkey.mail.satindar.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Mailersend DKIM",
+  "owner": {
+    "username": "satindar31",
+    "email": "satindar31@tutanota.com"
+  },
+  "record": {
+    "CNAME": "mlsend2._domainkey.mailersend.net"
```

Comment:
I don't think you can have underscores in CNAME records as far as I can recall.

Reply:
Can CNAME records not have underscores or does is-a.dev not support it. because if not, I can't use SMTP.